fem/bilinearform.hpp

```diff
@@ -1059,6 +1070,9 @@ class DiscreteLinearOperator : public MixedBilinearForm
    /// Adds a domain interpolator. Assumes ownership of @a di.
    void AddDomainInterpolator(DiscreteInterpolator *di)
    { AddDomainIntegrator(di); }
+   void AddDomainInterpolator(DiscreteInterpolator *di,
+                              Array<int> &elem_marker)
+   { AddDomainIntegrator(di, elem_marker); }
```

Comment:
`DiscreteLinearOperator::Assemble` does not check the markers. Should that check be added?

Reply:
Good catch, fixed in [052b5e0](https://github.com/mfem/mfem/pull/3524/commits/052b5e0cf20d35216e7dfb87d538bb4aceca074e). Thanks!